pkg/generate/app/env.go

```diff
@@ -24,6 +30,26 @@ func ParseEnvironment(vals ...string) Environment {
 	return env
 }
 
+func ParseEnvironment(vals ...string) (Environment, []string, []error) {
```

Comment:
This looks like it would accepts invalid things for env keys. What happens if you do for example 

```--env="S P A C E S=test"```

or have that pair in the file? Env vars are very strict about what's accepted for names, since you still have `validArgumentEnvironment`, make use of it.